### PR TITLE
Fix : export on bookkeeping

### PR DESCRIPTION
**Pull request by aspangaro**

### CODE REVIEW

File: htdocs/accountancy/journal/purchasesjournal.php [hunk 1]
```diff
@@ -210,9 +210,9 @@
         			$bookkeeping->doc_type = 'supplier_invoice';
         			$bookkeeping->fk_doc = $key;
         			$bookkeeping->fk_docdet = 0;    // Useless, can be several lines that are source of this record to add
-        			$bookkeeping->code_tiers = $tabcompany[$key]['code_fournisseur'];
+        			$bookkeeping->code_tiers = $tabcompany[$key]['code_compta_fournisseur'];
```
Comment (eldy) [inline htdocs/accountancy/journal/purchasesjournal.php:213]: the code_tiers is the customer code here. The accounting account is inside numero_compte

File: htdocs/accountancy/journal/purchasesjournal.php [hunk 2]
```diff
@@ -210,9 +210,9 @@
...
         			$bookkeeping->label_compte = dol_trunc($companystatic->name, 16) . ' - ' . $invoicestatic->refsupplier . ' - ' . $langs->trans("Code_tiers");
-        			$bookkeeping->numero_compte = $tabcompany[$key]['code_compta_fournisseur'];
+        			$bookkeeping->numero_compte = $conf->global->ACCOUNTING_ACCOUNT_SUPPLIER;
```
Comment (eldy) [inline htdocs/accountancy/journal/purchasesjournal.php:215]: Must be the value defined during the bind step. Why changing this ?

**Comment by aspangaro** [comment]
After binding, when you write in the general ledger, we must have 401000 in account_number & FELDY in account_thirdparty for exemple but today we have FELDY in account_number and dolibarr supplier number in account_thirdparty.

A lot of accountancy software work that way. With a principal account and a thirdparty account, not with 2 thirdparty account, it's called "centralisation". This is the official functioning before Dolibarr V5.

We have 3 centralisation in Dolibarr, supplier (401000), customer (411000) and user (421000). For me, it's better for the future lettering function and especially for the FEC export who are the french standard export for the administration.

I don't verify in Dolibarr v5 but it's a major change if we have change to take this functionning.

**Comment by aspangaro** [comment]
ping @eldy

**Comment by eldy** [comment]
I agree, you should have 401000 into account_number (property numero_compte), so we must fill with
code_compta_fournseur set during the binding. That why we should not change current code. This is exactly what current code do in current version in 5.0 branch. With current version, things work like you describe, i can't reproduce pb. Do yoi have a environment (database dump) you can send me that does not work with current code. On my side it already works.

**Pull request closed without merging**